edweiss-app/app/(app)/quiz/lectureQuizStudentView.tsx

```diff
@@ -174,14 +98,7 @@ export const LectureQuizResultDisplay: ReactComponent<{ studentAnswer: QuizzesAt
 			<TSafeArea>
 				<TView>
 					<TFResultDisplay key={exercise.question + "result"} exercise={exercise} selected={studentAnswer.value as boolean | undefined} result={result.value as boolean} />
```

Comment:
`... + "result"` adding "result" is unnecessary as each key must be unique, it's invariant across all elements.

Reply:
You are right, this was more about clarity.